snap-core/src/main/java/org/esa/snap/core/datamodel/TiePointGeoCoding.java

```diff
@@ -776,6 +802,18 @@ public boolean transferGeoCoding(final Scene srcScene, final Scene destScene, fi
         }
     }
 
+    protected void copyGridsToDestScene(Scene destScene) {
+        final Product destProduct = destScene.getProduct();
+        final TiePointGrid lonGrid = getLonGrid();
+        if (!destProduct.containsTiePointGrid(lonGrid.getName())) {
+            destProduct.addTiePointGrid(lonGrid);
```

Comment:
Can this be the right way?
A TiePointGrid is a ProductNode and has an Owner field.
If a ProductNode is assigned to the destProduct without cloning the node, the Owner referes the destProduct but the Node is still part of the source product. This leads to inconsistency.

Reply:
correct, done